testnet/stacks-node/src/tests/signer/commands/context.rs

```diff
@@ -0,0 +1,59 @@
+use crate::tests::signer::v0::MultipleMinerTest;
+use madhouse::{State, TestContext};
+use std::fmt::Debug;
+use std::sync::{Arc, Mutex};
+use std::time::Duration;
+
+pub struct SignerTestContext {
+    pub miners: Arc<Mutex<MultipleMinerTest>>,
+}
+
+impl Debug for SignerTestContext {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("SignerTestContext").finish()
+    }
+}
+
+impl Clone for SignerTestContext {
```

Comment:
nit: `#[derive(Clone)]` instead

Reply:
Addressed in https://github.com/stacks-network/stacks-core/pull/6007/commits/6979eb1e66484291361dddb1c51fdb45d95defe2.